tests/test_git.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+import subprocess
+
+import sand_env
+
+from pubs import config
+
+
+def git_hash(pubsdir):
+    """Return the git revision"""
+    hash_cmd = ('git', '-C', pubsdir, 'rev-parse', 'HEAD')
+    return subprocess.check_output(hash_cmd)
+
+
+class TestGitPlugin(sand_env.SandboxedCommandTestCase):
+
+    def setUp(self, nsec_stat=True):
+        super(TestGitPlugin, self).setUp()
+        self.execute_cmds([('pubs init',)])
+        conf = config.load_conf(path=self.default_conf_path)
+        conf['plugins']['active'] = ['git']
+        config.save_conf(conf, path=self.default_conf_path)
+
+    def test_git(self):
```

Comment:
As an alternative we could check the number of lines on a git one-line log. But this is fine for now too.

Reply:
Yep. I will leave that as an exercise for later.